kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionSerializer.java

```diff
@@ -360,7 +360,7 @@ private void handleSessionSerialization(String sessionId,
             unrecoverableError = true;
         } catch (IOException e) {
             getLogger().warn(
-                    "Optimistic serialization of session {} with distributed key {} failed",
+                    "Optimistic serialization of session {} with distributed key {} failed. Switching to pessimistic locking.",
```

Comment:
Updated the log message as it is for other cases when switching to pessimistic locking, or should we handle the `IOException` as unrecoverable error too?

Reply:
I would say that an IOException is unrecoverable, and it probably does not make sense to make an additional attempt, since it will most likely fail again.
There's a little chance that the error is temporary, but I would go with the `unrecoverable` flag.